### DISCUSSION TITLE
Product reviews app

### DISCUSSION THREAD

**Discussion by krzysztofwolski** [saleor/apps#747] [Integrations & features]
# Scope
- synchronize products with external service
- [optional] update rating field in `Product`, which can be used later for sorting products
- [refinement needed] sending emails with rating requests after completed orders

# Example providers:
- https://trust.reviews/
- https://judge.me/

**Comment by jayelkaake** [comment]
We'll build a [Fera](https://www.fera.ai) integration if enough people want it. It has everything a merchant would need for reviews.

**Reply by lkostrowski** [reply]
Sounds good. Feel free to reach us for any support
